reference/3.0/PSDiagnostics/Disable-PSWSManCombinedTrace.md

```diff
@@ -1,44 +1,62 @@
 ---
 external help file: PSDiagnostics-help.xml
+keywords: powershell,cmdlet
+locale: en-us
 Module Name: PSDiagnostics
-online version:
+ms.date: 11/29/2018
 schema: 2.0.0
```

Comment:
Does "Help Version" also need to be added to these?  It was added to the other files in this module.  Please validate for all files updated in PR.